package.json

```diff
@@ -96,8 +96,8 @@
     "@babel/runtime": "^7.0.0",
     "focus-lock": "^0.9.1",
     "prop-types": "^15.6.2",
```

Comment:
I'm wondering if we should move `prop-types` to peer dependency as well 🤔

Reply:
No. The [general recommendation](https://github.com/facebook/prop-types#how-to-depend-on-this-package) is to have it in deps.